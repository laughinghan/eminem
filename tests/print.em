let .x 1
let .y "lol"
print "x: " x ", y:" y "
"

println.on.many.lines.line.after.line
println ^a b c()
