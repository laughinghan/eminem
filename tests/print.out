x: 1, y:lol
on
many
lines
line
after
line
